app/components/PaginatedEventList.tsx

```diff
@@ -29,15 +29,18 @@ const PaginatedEventList = React.memo<Props>(function PaginatedEventList({
       heading={heading}
       fetch={fetch}
       options={options}
-      renderItem={(item, index) => (
-        <EventListItem
-          key={item.id}
-          event={item}
-          document={document}
-          latest={index === 0}
-          {...rest}
-        />
-      )}
+      renderItem={(item, index, composite) => {
+        return (
+          <EventListItem
+            key={item.id}
+            event={item}
+            document={document}
+            latest={index === 0}
+            composite={composite}
+            {...rest}
+          />
+        );
+      }}
```

Comment:
```suggestion
      renderItem={(item, index, props) => {
        return (
          <EventListItem
            key={item.id}
            event={item}
            document={document}
            latest={index === 0}
            {...props}
            {...rest}
          />
        );
      }}
```

Prefer?

Reply:
I can go either way — 
- `props` is nice but very generic and doesn't give you a hint as to what it's for. if you don't need keyboard nav, you can actually not include this
- `composite` breaks a bit of abstraction but I think in an instructive way
- `compositeItemProps` is the actually most self-documenting but it's... clunky

happy to defer to you on this one - what would you prefer?

Reply:
`compositeItemProps` and spread would be my preferred, but again It's not a hill I'm going to die on.

Reply:
ok, changed to `compositeProps` and spread